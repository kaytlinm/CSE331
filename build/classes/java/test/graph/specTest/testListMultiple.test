#List Multiple Test File
#Tests that a graph can be created, multiple (3) nodes can be added and printed out.

CreateGraph graph1
AddNode graph1 a
AddNode graph1 b
AddNode graph1 c
ListNodes graph1
