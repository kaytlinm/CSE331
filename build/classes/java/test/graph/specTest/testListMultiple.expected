#List Multiple Test File
#Tests that a graph can be created, multiple (3) nodes can be added and printed out.

created graph graph1
added node a to graph1
added node b to graph1
added node c to graph1
graph1 contains: a b c